editor.planx.uk/src/pages/FlowEditor/components/Sidebar/Customisations/CustomisationCard.tsx

```diff
@@ -34,6 +34,7 @@ export const CustomisationCard: React.FC<Props> = ({
         ComponentType.Checklist,
         ComponentType.ResponsiveQuestion,
         ComponentType.ResponsiveChecklist,
+        ComponentType.InternalPortal,
```

Comment:
This line addresses an adjacent bug report https://opensystemslab.slack.com/archives/C04DZ1NBUMR/p1750877038481739?thread_ts=1750424102.836949&cid=C04DZ1NBUMR: 
> There's also an issue with how internal portal Allow edit toggles are treated in the 'Customise' tab. They appear as customisation of the portal itself (i.e. the name/reference), rather than customisations within that portal.

Now, we'll mark an internal portal as "complete" if any children have been added to it (in addition to original behavior of if it's been directly edited/renamed etc)